docs/howto/krustlet-on-lke.md

```diff
@@ -0,0 +1,281 @@
+  --label=${LABEL} \
+  --authorized_users=${USER}
+```
+
+You may then grab the Linode's `ID` and public IP `IP` using:
+
+```console
+$ LINODE_ID=$(linode-cli linodes list \
+  --label=${LABEL} \
+  --format=id \
+  --no-headers \
+  --text) && echo ${LINODE_ID}
+
+$ IP=$(linode-cli linodes list \
+  --label=${LABEL} \
+  --format=ipv4 \
+  --no-headers \
+  --text) && echo ${IP}
+```
+
+## Step 2: Get a bootstrap config for your Krustlet node
+
+Krustlet requires a bootstrap token and config the first time it runs. Follow
+the guide [here](bootstrapping.md), setting the `CONFIG_DIR` variable to `./`,
+to generate a bootstrap config and then return to this document. If you already
+have a kubeconfig available that you generated through another process, you can
+proceed to the next step. However, the credentials Krustlet uses must be part of
+the `system:nodes` group in order for things to function properly.
+
+NOTE: You may be wondering why you can't run this on the VM you just
+provisioned. We need access to the Kubernetes API in order to create the
+bootstrap token, so the script used to generate the bootstrap config needs to be
+run on a machine with the proper Kubernetes credentials.
+
+## Step 3: Copy bootstrap config to VM
+
+The first thing we'll need to do is copy up the assets we generated in steps 1
+and 2. Copy them to the Linode by typing:
+
+```console
+$ scp \
+  -i ${PRIVATE_KEY} \
+  ${HOME}/.krustlet/config/bootstrap.conf \
+  root@${IP}:.
+```
+
+> **NOTE** You'll need to set `PRIVATE_KEY` to the path to the private key
+corresponding to the SSH key pair that you used to create the Linode.
+
+We can then SSH into the Linode by typing:
+
+```console
+$ ssh \
+  -i ${PRIVATE_KEY} \
+  root@${IP}
+```
+
+## Step 4: Install and configure Krustlet
+
+Install the latest release of krustlet following [the install
+guide](../intro/install.md).
+
+There are two flavors of Krustlet (`krustlet-wasi` and `krustlet-wascc`), let's
+use the first:
+
+```console
+$ KUBECONFIG=${PWD}/kubeconfig krustlet-wasi \
+--hostname="krustlet" \
+--node-ip=${IP} \
+--node-name="krustlet" \
+--bootstrap-file=./bootstrap.conf \
+--cert-file=${PWD}/krustlet.crt \
+--private-key-file=${PWD}/krustlet.key
+```
+
+**NOTE** To increase the level of debugging, you may prefix the command with
+`RUST_LOG=info` or `RUST_LOG=debug`.
+
+**NOTE** The value of `${IP}` was determined in step #1.
+
+### Step 4a: Approving the serving CSR
+
+Once you have started Krustlet, there is one more manual step (though this could
+be automated depending on your setup) to perform. The client certs Krustlet
+needs are generally approved automatically by the API. However, the serving
+certs require manual approval. To do this, you'll need the hostname you
+specified for the `--hostname` flag or the output of `hostname` if you didn't
+specify anything. From another terminal that's configured to access the cluster,
+run:
+
+```console
+$ kubectl certificate approve krustlet-tls
```

Comment:
I would change `krustlet-tls` to be `$HOSTNAME` so people know to use the hostname as discovered in the paragraph above

Reply:
Fixed: https://github.com/deislabs/krustlet/pull/497/commits/43d4006a95f98fe968e1ec9f084b28b69dd404a3